rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/TaskControllerIntTest.java

```diff
@@ -83,6 +82,7 @@ public void testGetAllTasks() {
         ResponseEntity<PagedResources<TaskSummaryResource>> response = template.exchange(
             "http://127.0.0.1:" + port + "/v1/tasks", HttpMethod.GET, request,
             new ParameterizedTypeReference<PagedResources<TaskSummaryResource>>() {
+
```

Comment:
this line is superfluous, right?

Reply:
Yes, I remove all this same lines in this file